app/assets/javascripts/volunteers.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
How would you feel about removing this file and re-creating if you'd like to add some javascript here?